config/shared/errors/errors.go

```diff
@@ -74,6 +74,7 @@ var (
 	// Systemd section errors
 	ErrInvalidSystemdExt       = errors.New("invalid systemd unit extension")
 	ErrInvalidSystemdDropinExt = errors.New("invalid systemd drop-in extension")
+	ErrNoDotSuffix             = errors.New("a dot isn't present in the systemd unit string")
```

Comment:
For consistency with `ErrInvalidSystemdExt`, what about `ErrNoSystemdExt` = `no systemd unit extension`?

Reply:
Yeah, that makes sense. Let me change the message.